ios/FluentUI/Table View/TableViewTokens.swift

```diff
@@ -0,0 +1,167 @@
+    open var cellBackgroundGrouped: DynamicColor {
+        .init(light: aliasTokens.backgroundColors[.neutral1].light,
+              dark: aliasTokens.backgroundColors[.neutral2].dark)}
+
+    /// The selected background color of the TableViewCell.
+    open var cellBackgroundSelectedColor: DynamicColor { aliasTokens.backgroundColors[.neutral5] }
+
+    /// The leading image color.
+    open var imageColor: DynamicColor { aliasTokens.foregroundColors[.neutral1] }
+
+    /// The size dimensions of the customView.
+    open var customViewDimensions: CGSize {
+        switch customViewSize {
+        case .zero:
+            return .zero
+        case .small:
+            return CGSize(width: globalTokens.iconSize[.medium],
+                          height: globalTokens.iconSize[.medium])
+        case .medium, .default:
+            return CGSize(width: globalTokens.iconSize[.xxLarge],
+                          height: globalTokens.iconSize[.xxLarge])
+        }
+    }
+
+    /// The trailing margin of the customView.
+    open var customViewTrailingMargin: CGFloat {
+        switch customViewSize {
+        case .zero:
+            return globalTokens.spacing[.none]
+        case .small:
+            return globalTokens.spacing[.medium]
+        case .medium, .default:
+            return globalTokens.spacing[.small]
+        }
+    }
+
+    /// The title label color.
+    open var titleColor: DynamicColor { aliasTokens.foregroundColors[.neutral1] }
+
+    /// The subtitle label color.
+    open var subtitleColor: DynamicColor { aliasTokens.foregroundColors[.neutral3] }
+
+    /// The footer label color.
+    open var footerColor: DynamicColor { aliasTokens.foregroundColors[.neutral3] }
+
+    /// The color of the selectionImageView when it is not selected.
+    open var selectionIndicatorOffColor: DynamicColor { aliasTokens.foregroundColors[.neutral3] }
+
+    /// The font for the title.
+    open var titleFont: FontInfo { aliasTokens.typography[.body1] }
+
+    /// The font for the subtitle when the TableViewCell is two lines.
+    open var subtitleTwoLinesFont: FontInfo { aliasTokens.typography[.caption1] }
+
+    /// The font for the subtitle when the TableViewCell is three lines.
+    open var subtitleThreeLinesFont: FontInfo { aliasTokens.typography[.body2] }
+
+    /// The font for the footer.
+    open var footerFont: FontInfo { aliasTokens.typography[.caption1] }
+
+    /// The leading margin for the labelAccessoryView.
+    open var labelAccessoryViewMarginLeading: CGFloat { globalTokens.spacing[.xSmall] }
+
+    /// The trailing margin for the labelAccessoryView.
+    open var labelAccessoryViewMarginTrailing: CGFloat { globalTokens.spacing[.xSmall] }
+
+    /// The leading margin for the customAccessoryView.
+    open var customAccessoryViewMarginLeading: CGFloat { globalTokens.spacing[.xSmall] }
+
+    /// The minimum vertical margin for the customAccessoryView.
+    open var customAccessoryViewMinVerticalMargin: CGFloat = 6
```

Comment:
This also needs to have #1044 applied

Reply:
Thanks for catching it. Just cherry-picked that change.